analyzers/tests/SonarAnalyzer.UnitTest/Rules/SymbolicExecution/SymbolicExecutionRunnerTest.cs

```diff
@@ -526,7 +538,16 @@ public class Sample
 {{
     {members}
 }}";
-        VerifyCode<TestSERunnerCS>(code, ProjectType.Product, ParseOptionsHelper.FromCSharp9, null, MainScopeAssignmentRuleCheck.SMain, BinaryRuleCheck.SBinary);
+        VerifyCode<TestSERunnerCS>(code, ProjectType.Product, ParseOptionsHelper.FromCSharp9, null, false, MainScopeAssignmentRuleCheck.SMain, BinaryRuleCheck.SBinary);
+    }
+
+    private static void VerifyTopLevelStatements(string members)
```

Comment:
I'm wondering if we need this to test other top-level statement cases in the future. WDYT?

Reply:
I just used it in two tests - so I guess the answer is yes.